x-pack/plugins/canvas/kibana.json

```diff
@@ -13,10 +13,12 @@
       "expressionError",
       "expressionRevealImage",
       "expressions",
+      "expressionShape",
       "features",
       "inspector",
       "presentationUtil",
-      "uiActions"
+      "uiActions",
+      "expressionRevealImage"
```

Comment:
Reveal Image is duplicated here.  Probably bad conflict resolution.

Reply:
Yes, this rebase was horrible.)